libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/SyncUpTarget.java

```diff
@@ -190,7 +207,7 @@ public int deleteOnServer(SyncManager syncManager, JSONObject record) throws JSO
      * @return server response status code
      * @throws IOException
      */
-    public int deleteOnServer(SyncManager syncManager, String objectType, String objectId) throws IOException {
+    protected int deleteOnServer(SyncManager syncManager, String objectType, String objectId) throws IOException {
```

Comment:
That method used to be called from SyncManager
Now SyncManager calls the target.deleteOnServer(syncManager, record) instead which calls deleteOnServer(syncManager, objectType, objectId).
SyncManager no longer assumes that record are SObject (i.e. with type and id).
This method is now protected because it's only called from the target object itself.